### PR TITLE
feature #3248 - new contact profile

### LINKED ISSUE #3248: Rework Contact profile details UI
### User Story

As a user, I want a clean profile screen for user I chat with

Designs: https://zpl.io/2pxZQol (7th and 8th from left)

Type: Feature

### PR DESCRIPTION
**Pull request by goranjovic**
fixes #3248 

### Summary:

Updated contact profile screen, see #3248

### Steps to test:
- Start chatting and reach contact profile screen

status: ready

### CODE REVIEW

**Review by jeluard — APPROVED** [review]

**Comment by jeluard** [comment]
@goranjovic Is this still WIP?

**Comment by goranjovic** [comment]
@jeluard No, it's fine. Just checked it on ios too - looks good.
The only issue is that until #3547 is merged this screen won't have correct data for contacts not in your
contact list.

**Comment by goranjovic** [comment]
Android: 

screenshot-2018-03-15_08 10 41 537

ios:

img_4903

**Review by dmitryn — APPROVED** [review]

**Review by cammellos — APPROVED** [review]

File: resources/icons/add_contact.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+    <g fill="none" fill-rule="evenodd">
+        <path fill="#000000" fill-rule="nonzero" d="M16.09 19.727v-1.636a4.273 4.273 0 0 0-4.272-4.273H5.273A4.273 4.273 0 0 0 1 18.091v1.636a1 1 0 1 0 2 0v-1.636a2.273 2.273 0 0 1 2.273-2.273h6.545a2.273 2.273 0 0 1 2.273 2.273v1.636a1 1 0 1 0 2 0zm-7.545-9.182a2.273 2.273 0 1 0 0-4.545 2.273 2.273 0 0 0 0 4.545zM18 11V9.003c0-.554.647-1.003.926-1.003.276 0 .989.438.989 1.003V11h1.78a.999.999 0 0 1 1.002 1c0 .552-.438 1-1.003 1h-1.78v2c0 .554-.71 1.003-.988 1.003-.277 0-.926-.438-.926-1.003v-2h-1.997A.999.999 0 0 1 15 12c0-.552.438-1 1.003-1H18zm-9.455 1.545a4.273 4.273 0 1 1 0-8.545 4.273 4.273 0 0 1 0 8.545z"/>
```
Comment (flexsurfer) [inline resources/icons/add_contact.svg]: please remove fill value to do not confuse, it's not used

File: resources/icons/in_contacts.svg [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
+    <g fill="none" fill-rule="evenodd">
+        <path fill="#000000" fill-rule="nonzero" d="M16.09 19.727v-1.636a4.273 4.273 0 0 0-4.272-4.273H5.273A4.273 4.273 0 0 0 1 18.091v1.636a1 1 0 1 0 2 0v-1.636a2.273 2.273 0 0 1 2.273-2.273h6.545a2.273 2.273 0 0 1 2.273 2.273v1.636a1 1 0 1 0 2 0zm-7.545-9.182a2.273 2.273 0 1 0 0-4.545 2.273 2.273 0 0 0 0 4.545zm8.38 1.115l4.368-4.367a1 1 0 0 1 1.414 1.414L17.716 13.7a1 1 0 0 1-1.35.075l-2.49-1.993a1 1 0 1 1 1.249-1.562l1.8 1.441zm-8.38.885a4.273 4.273 0 1 1 0-8.545 4.273 4.273 0 0 1 0 8.545z"/>
```
Comment (flexsurfer) [inline resources/icons/in_contacts.svg]: please remove fill value to do not confuse, it's not used

**Review by flexsurfer — APPROVED** [review]

**Comment by lukaszfryc** [comment]
I've tested it on Galaxy S8 and iPhone 6. Things I've noticed:

1. iOS only: The profile screen can be scrolled which probably does not make sense since everything is visible. I'd have scroll only when it's needed (content does not fit the screen)

2. Would be nice to be able to copy contact code in order to share it with others

3. Android only **(issue present in nightly as well)**: When „send transaction” button is clicked, the „/send” command is put into input but it does not trigger keyboard to open

@goranjovic @jeluard I think we can close this PR and think about 1), 2) later. WDYT?

**Comment by goranjovic** [comment]
@lukaszfryc  Makes sense

**Comment by lukaszfryc** [comment]
Note that the 2) is a regression. Previously you could copy and (separate option) share the contact code.

Let's say I meet someone new in a public or group chat. Now, there is no way for him to share his contact code so I can add it :(

@denis-sharypin is it intentional that we got rid of it?

**Comment by denis-sharypin** [comment]
@lukaszfryc @goranjovic we got rid off three dots near contact code but we need to introduce a platform native way of selecting and coping those codes. It could be done as a seperate issue. I’ll create it.

**Comment by lukaszfryc** [comment]
Cool. I'll create tickets for 1) and 3).

@goranjovic I think you can merge now

**Comment by lukaszfryc** [comment]
@goranjovic I've also noticed that option to remove a contact is now gone. I created #3585

**Pull request merged**